store/actions/alert.actions.ts

```diff
@@ -1,9 +1,7 @@
-import { Dispatch } from "redux";
-import { v4 as uuid } from "uuid";
 import { REMOVE_ALERT, SET_ALERT } from "../types/alert.types";
 
 export const setAlert = (msg: string, alertType: string) => (dispatch: any) => {
-  const id = uuid();
+  const id = (Math.floor(Math.random() * 100) + Date.now()).toString();
```

Comment:
Create generateId function 

```suggestion
  const id = generateId();
```

Reply:
Created a service for this